error-prone-contrib/src/test/java/tech/picnic/errorprone/bugpatterns/DirectReturnTest.java

```diff
@@ -223,4 +223,18 @@ void replacement() {
             "}")
         .doTest(TestMode.TEXT_MATCH);
   }
+
+  @Test
+  void ignoreClassesAnnotatedWithLombokData() {
+    CompilationTestHelper.newInstance(DirectReturn.class, getClass())
+        .addSourceLines(
+            "A.java",
+            "import lombok.Data;",
+            "",
+            "@Data",
+            "class A {",
+            "  private String field;",
+            "}")
+        .doTest();
+  }
```

Comment:
If I revert the fix then this test doesn't fail. Contrary to what's claimed further up, as best as I can tell Lombok is _not_ executed here. Seems we need to explicitly enable the annotation processor:
```
.setArgs("-processor", "lombok.launch.AnnotationProcessorHider$AnnotationProcessor")
```